Eastern-Research-Group/mywaterway
# Feature/98 refactor public index pages

## Related Issues:
* https://jira.epa.gov/browse/HMW-98
* https://jira.epa.gov/browse/HMW-134

## Main Changes:
* Cleaned up some of the code in the public index.html file.
* Switched from `document.location` to `window.location` across the app and cypress tests.
* Removed Internet Explorer specific code. 
  * While I was working on this ticket I went to test the app in IE and noticed that it did not load at all (syntax errors). I did some more research and found that Esri ended support for IE as of version 4.18. Esri says that apps using ArcGIS JS API will no longer work in IE. For more information see the following links: [ArcGIS System Requirements](https://developers.arcgis.com/javascript/latest/system-requirements/#:~:text=Supported%20browsers&text=Firefox%2093%20or%20later,Safari%2014%20or%20later) and [So Long Internet Explorer](https://www.esri.com/arcgis-blog/products/arcgis/announcements/so-long-internet-explorer-11/)
* Added an error message, only visible to IE users, that says the IE is not supported.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/overview in a non IE browser
2. Verify the app still works
3. Verify there is not a message about IE not being supported
4. Open an IE browser 
5. Verify there is a message saying that IE is not supported

Note: I did steps 4 and 5 using IE 11, 10, 9, and 8 with browserstack and the message shows up correctly for all except for IE 8. None of the styles loaded in for IE 8, but [epa.gov](https://www.epa.gov/) has the same problem.
